### PR TITLE
added width adjustment input box to Editor Preferences menu

### PR DESCRIPTION
**Pull request by infinitelydense**
Recreated [PR 19](https://github.com/lmg-anon/mikupad/pull/19) from a separate branch. Hopefully it looks good this time, please let me know if anything else needs fixing.

### CODE REVIEW

File: mikupad.html [hunk 1]
```diff
@@ -22,6 +22,10 @@
 </script>
 <style>
 
+:root {
+	--prompt-container-width: 36em;
+	}
+
```
Comment (lmg-anon) [inline mikupad.html:28]: Is there a reason to use `:root` instead of the `html` one below?

File: mikupad.html [hunk 2]
```diff
@@ -160,8 +164,8 @@
 }
 @media (min-width: calc(40.5em + 250px)) {
```
Comment (lmg-anon) [inline mikupad.html:165]: You also need to change this media rule and the one above, this should be the code that you need to use:
```css
@media (min-width: calc(calc(var(--prompt-container-width) * 1.125rem) + 250px))
```

**Review by lmg-anon — CHANGES_REQUESTED** [review]

**Comment by infinitelydense** [comment]
No reason it couldn't be in the html instead of root, that would be better. When it came to updating the media rule, using the prompt-container-width variable as described caused the sidebar to attach incorrectly. 

I think this project is awesome, but judging from my accidental closing of this request, I think I'll need to learn how to use git a bit better before contributing. 😅 I'll just leave this closed to avoid any further inconvenience.

**Comment by lmg-anon** [comment]
Don`t worry, it's not an inconvenience. Contributions are always welcome, and I can see why this is a useful feature.
When you feel comfortable to continue, feel free to reopen the PR.

**Pull request closed without merging**